### PR TITLE
Thruk 2.06 (new formula)

### PR DESCRIPTION
**Pull request by sgerrand**
:information_desk_person: These changes add a formula for [Thruk](www.thruk.org).

- [x] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

**Comment by sgerrand** [comment]
:memo: The Perl modules are a candidate for bottling.

File: Library/Formula/thruk.rb [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  homepage "http://www.thruk.org/"
+  url "https://github.com/sni/Thruk/archive/v2.06.tar.gz"
+  sha256 "85cd8d44a296887ab3245bfb2d988f4dfe4922f6d85848b05d6d5a5867a8c3d1"
+
+  depends_on "autoconf"   => :build
+  depends_on "automake"   => :build
+  depends_on "cpanminus"      => :build
+  depends_on "libtool"    => :build
+  depends_on "nagios"     => :recommended
+  depends_on "pkg-config" => :build
```
Comment (bfontaine) [inline Library/Formula/thruk.rb]: Please use only one space before `=>`.

File: Library/Formula/thruk.rb [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                          "--with-logdir=#{prefix}/var/log"
+    system "perl", "Makefile.PL"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/nagexp", "--version"
+    system "#{bin}/naglint", "--version"
+    system "#{bin}/thruk", "--version"
+  end
```
Comment (bfontaine) [inline Library/Formula/thruk.rb]: Can’t we write a test that does something more substantial than e.g. `--version` or `--help`?

**Comment by apjanke** [comment]
Closing this due to inactivity/lack of response. If you'd still like to pursue this, feel free to respond to the previous comments/questions and re-open on the new [`homebrew/core` repo](https://github.com/homebrew/homebrew-core).

**Pull request closed without merging**